sbin/shipper_utils/update_and_install_workers.py

```diff
@@ -155,7 +154,7 @@ def parse_arguments():
 
 def update_machine(machine,stats,args):
 
-    print(f"UPDATE MACHINE: {machine}")
+    os.system(f"echo UPDATE MACHINE: {machine}")
```

Comment:
Why change this from `print` to using `os.system`?

Reply:
The change here is due to the linux command being executed before the print statement here by the OS (or so I believe). By changing the print to an os.system, this seems to ensure that the `UPDATE MACHINE` line is printed before everything to ensure the log parsing is done properly.

Reply:
Yeah, this is because of how python works where it buffers text before writing it out to `stdout` and `stderr` so that the prints more efficient at the cost that it doesn't necessarily output immediately. `os.system` circumvents this buffering because of how it calls down to the underlying OS, at the expense that we also cannot do anything with its output or know that it errored really.

The better solution would be to make sure we just don't use `os.system` anywhere, and instead [`subprocess.run`](https://docs.python.org/3/library/subprocess.html#subprocess.run) (or equivalent `subprocess` method) which has the added benefit that it forces us to consider error cases where a command fails for some reason. This will also help us avoid these sorts of race conditions with prints when we're trying to parse the outputted log file.